FirebaseStorage/Tests/Unit/FIRStorageTests.m

```diff
@@ -16,7 +16,10 @@
 
 #import "FirebaseStorage/Sources/Public/FirebaseStorage/FIRStorageReference.h"
 
-#import "FirebaseStorage/Sources/FIRStorageComponent.h"
+#ifdef TODO
+// Port the component tests to Swift
+
+//#import "FirebaseStorage/Sources/FIRStorageComponent.h"
```

Comment:
Same here, should this commented out import be removed?
 ```suggestion
```

Reply:
This whole file needs to be ported to Swift